configure.ac

```diff
@@ -2988,7 +3004,7 @@ AC_MSG_RESULT($with_system_expat)
 # Check for use of the system libffi library
 AC_MSG_CHECKING(for --with-system-ffi)
 AC_ARG_WITH(system_ffi,
-            AS_HELP_STRING([--with-system-ffi], [build _ctypes module using an installed ffi library]),,,)
+            AS_HELP_STRING([--with-system-ffi], [build _ctypes module using an installed ffi library, see Doc/library/ctypes.rst (default is system dependant)]),,,)
```

Comment:
"system-dependent" (in the rest of the file, too). In British English, "dependant" and "dependent" have subtly different meanings (and this one is "dependent"); in US English, it's always "dependent".